src/component-library/theme/theme.ts

```diff
@@ -435,7 +435,7 @@ const theme = {
         marginBottom: 'var(--spacing-2)'
       },
       body: {
-        paddingY: 'var(--spacing-3)',
+        paddingY: 'var(--spacing-6)',
```

Comment:
Do not change here. Change on the `Select.style.tsx` instead line 71.

There should be:
```
 padding: 0 ${theme.dialog.medium.body.paddingX} ${theme.dialog.medium.body.paddingX} ${theme.dialog.medium.body.paddingX};
 ```